tamr_client/project.py

```diff
@@ -55,5 +64,55 @@ def _from_json(url: URL, data: JsonDict) -> Project:
         return mastering_project._from_json(url, data)
     elif proj_type == "CATEGORIZATION":
         return categorization_project._from_json(url, data)
+    elif proj_type == "SCHEMA_MAPPING_RECOMMENDATIONS":
+        return schema_mapping_project._from_json(url, data)
     else:
         raise ValueError(f"Unrecognized project type '{proj_type}' in {repr(data)}")
+
+
+def _create(
+    session: Session,
+    instance: Instance,
+    name: str,
+    project_type: str,
+    description: Optional[str] = None,
+    external_id: Optional[str] = None,
+    unified_dataset_name: Optional[str] = None,
+) -> Project:
+    """Create a project in Tamr.
+    Args:
```

Comment:
Needs whitespace for doc formatting:

```suggestion

    Args:
```

Reply:
updated

Reply:
this doesnt show up in the docs since its an internal function